.github/workflows/Nightly_Perf_Env_Odd_CI.yml

```diff
@@ -0,0 +1,261 @@
+# Nightly CI https://github.com/marketplace/actions/deploy-nightly
```

Comment:
BTW, could you arrange for the undesired workloads to be skipped depending upon the day of the week?  Something like

```
case "$(date +%w)" in
   0|3|5): ... ;;
   2|4|6): ... ;;
   *) skip all of the vdbench runs ;;
esac
```
so you don't need the second file that has to be kept in sync?